### DISCUSSION TITLE
What is this?

### DISCUSSION THREAD

**Discussion by benthillerkus** [Discoin/discoin#1] [Q&A] [answered]
and could you please not?

**Comment by austinhuang0131** [comment, marked as answer]
> What is this?

~~discord/discoin is probably a joke.~~ EDIT: The repository was deleted

We are an API that allows participating Discord bots to exchange their bot currencies to one another. It was intended to simulate a "stock market" on Discord, but it faces low adoption. We have been in existence for 3 years or so.

> and could you please not?

This organization has been there for 3 years or so, and we have various code repositories. It's fair game. Plus I have indicated the correct link on README.md, and Discord should be (and if not, already is) somewhat aware of our existence (through the bot verification form).

**Reply by benthillerkus** [reply]
aight
I was afraid discord was starting their own cryptocurrency or smth like that because of discord/discoin
